@types/ol-ext/interaction/Transform.d.ts

```diff
@@ -144,9 +152,14 @@ export default class Transform extends Pointer {
   setActive(b: boolean): void;
 
   /**
-   * Set efault sketch style
+   * Set default sketch style
+   * @param {OlExtTransformDefaultStyle} options Style options for the default style
+   * @param {Stroke} options.stroke Stroke style for selection rectangle
+   * @param {Fill} options.fill Fill style for selection rectangle
+   * @param {Stroke} options.pointStroke Stroke style for handles
+   * @param {Fill} options.pointFill Fill style for handles
    */
-  setDefaultStyle(): void;
+  setDefaultStyle(options: OlExtTransformDefaultStyle): void;
```

Comment:
```suggestion
  setDefaultStyle(options?: OlExtTransformDefaultStyle): void;
```
This must be a nullable/undefined type as well